airflow/www/templates/airflow/dags.html

```diff
@@ -399,10 +399,6 @@ <h2>{{ page_title }}</h2>
                       <span class="material-icons" aria-hidden="true">code</span>
                       Code
                     </a>
-                    <a href="{{ url_for('Airflow.dag_details', dag_id=dag.dag_id) }}" class="dags-table-more__link">
```

Comment:
Let's leave this and just change the URL

Reply:
U mean to change the url to `{{ url_for('Airflow.grid', dag_id=dag.dag_id, num_runs=num_runs) }}` i.e the dag detials in Grid view?

Reply:
Changed the URL to point to grid view of dag details